booking-service/src/test/java/co/axelrod/bookingservice/service/BookingServiceTest.java

```diff
@@ -63,6 +68,23 @@ void confirmBookingTest() {
             - Guest finished booking
             """)
     void happyPath() {
+        CreateBookingRequest request = new CreateBookingRequest();
+        Booking booking_from_request = mapper.map(request);
+        request.setGuestId("guest123");
+        request.setPropertyId("property456");
+        request.setTimestampFrom(System.currentTimeMillis());
+        request.setTimestampTo(request.getTimestampFrom() + 86400000);
```

Comment:
Do not use "magic strings". Introduce a variable for 86400000, something like:
private static final long ONE_DAY_IN_MS = 86400000